File: src/Infrastructure/Builder/Templates/CSharp/Robot.cs
using cAlgo.API;

namespace cAlgo.Robots;

[Robot(AccessRights = AccessRights.None)]
public partial class {IdentityClass} : Robot
{
    protected override void OnStart()
    {
        // To learn more about cTrader Algo visit our Help Center:
        // https://help.ctrader.com/ctrader-algo/

        Print("Hello, cBot!");
    }

    protected override void OnTick()
    {
        // Handle price updates here
    }

    protected override void OnStop()
    {
        // Handle cBot stop here
    }
}

File: src/Infrastructure/Builder/Templates/Python/Robot.py
import clr

clr.AddReference("cAlgo.API")

# Import cAlgo API types
from cAlgo.API import *

# Import trading wrapper functions
from robot_wrapper import *

class {IdentityClass}():
    def on_start(self):
        print(api.Message)
        # To learn more about cTrader Algo visit our Help Center:
        # https://help.ctrader.com/ctrader-algo/

    def on_tick(self):
        # Handle price updates here
        pass

    def on_stop(self):
        # Handle cBot stop here
        pass